lib/smart_answer/calculators/uk_benefits_abroad_calculator.rb

```diff
@@ -97,7 +97,7 @@ def former_yugoslavia?
 
     def social_security_countries_jsa?
       if going_abroad
-        (COUNTRIES_OF_FORMER_YUGOSLAVIA + %w[guernsey jersey new-zealand]).include?(country)
+        (COUNTRIES_OF_FORMER_YUGOSLAVIA + %w[guernsey new-zealand]).include?(country)
```

Comment:
Would be great to link to the actual commit in this commit message, instead of these directions `lib/smart_answer_flows/uk-benefits-abroad.rb#85 et seq.`. It'll only get harder to figure out how that file looked at the time, as time goes on!

(I personally use the GitLens extension for VS code, which makes this nice and easy. Right Click on the line -> Copy As -> Copy Remote File URL).